ustwo-clockwise/src/main/java/com/ustwo/clockwise/wearable/permissions/PermissionsRequestor.java

```diff
@@ -98,16 +98,16 @@ private void requestNextPermission() {
         }
     }
 
-    /**
-     * If there is a wearable permission request, we assume it's just a single permission in the request item list.
-     */
-    private void requestWearablePermission() {
-        if(mWearablePermissionToRequest.getPermissions().size() > 0) {
-            String permission = mWearablePermissionToRequest.getPermissions().get(0);
-            
+    private void requestNextWearablePermission() {
```

Comment:
method name is a bit confusing since it's also handles companion permission requests. could be solved either by renaming it or just adding a comment